force-app/main/default/lwc/propertyLocation/propertyLocation.js

```diff
@@ -27,6 +27,10 @@ export default class PropertyLocation extends LightningElement {
         }
     }
 
+    get hasNoErrors() {
```

Comment:
Not a big deal but instead of introducing a new JS getter, I would have switched the template order so that you have `<template lwc:if={errors}>` and `<template lwc:else>`.